### DISCUSSION TITLE
Problems in use[Feat]:

### DISCUSSION THREAD

**Discussion by chenhuanguang** [coder/code-server#5660] [Q&A]
The first time you start the code server, you can enter the directory selection page. After you select the page, the second time you come in, you will still select the directory you previously selected. Is there any way to select the directory page when you come in the first time; Is it the caching problem? How to clean it? How can I completely uninstall the code server in the form of a compressed package? Here are two ways I think
This is how I start the code server locally "./bin/code-server --port 8888 --host 0.0.0.0 --auth none"

I hope this page is always here
<img width="1401" alt="image" src="https://user-images.githubusercontent.com/56351544/196161818-a4589559-6077-4c13-93bd-65e80574769e.png">

**Comment by jsjoeio** [comment]
I believe using `--ignore-last-opened` should open an empty window each time.
